pkgs/development/python-modules/urlextract/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, appdirs
+, buildPythonPackage
+, dnspython
+, fetchPypi
+, filelock
+, idna
+, pytestCheckHook
+, uritools
+}:
+
+buildPythonPackage rec {
+  pname = "urlextract";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-yxOuiswFOJnAvxwTT++Zhk8nZWK2f4ePsQpUYI7EYS4=";
+  };
+
+  propagatedBuildInputs = [
+    appdirs
+    filelock
+    idna
+    uritools
+  ];
+
+  checkInputs = [
+    dnspython
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # fails with dns.resolver.NoResolverConfiguration due to network sandboxing
+    "test_check_dns_enabled"
+    "test_check_dns_find_urls"
+    "test_dns_cache_init"
+    "test_dns_cache_negative"
+    "test_dns_cache_reuse"
+  ];
+
+  pythonImportsCheck = [ pname ];
+
+  meta = with lib; {
+    description = "Collects and extracts URLs from given text.";
```

No period at the end.

```suggestion
    description = "Collects and extracts URLs from given text";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes